pkg/aspect/configError.go

```diff
@@ -77,5 +77,19 @@ func (e ConfigError) Error() string {
 }
 
 func (e *ConfigErrors) Error() string {
+	if e == nil || e.Multi == nil {
+		return ""
+	}
 	return e.Multi.Error()
 }
+
+// Extend joins 2 configErrors together.
+func (e *ConfigErrors) Extend(ee *ConfigErrors) *ConfigErrors {
```

Comment:
this, to me, sorta looks like we should just use `me.Error` instead of our own `ConfigErrors`